requirements.txt

```diff
@@ -1,5 +1,5 @@
 asgiref==3.5.2
-Django==4.1
+Django
```

Comment:
The version of the Project should not be changed. It is highly recommended to keep this. This is the version that is used on the current project. we have to put that back. if not it will cause conflict.